plugins/inventory/now.py

```diff
@@ -120,6 +120,12 @@
     type: bool
     default: false
     version_added: 1.3.0
+  experimental_aggregation:
```

Comment:
As we discussed a while ago, this matches the solution that we discussed doing - putting the new functionality behind a flag that will allow users to opt in to the new feature if they want to use it.

That said, I'm a little worried about calling this "experimental_aggregation", since at some point it's going to stop being experimental, and I don't want to change the API again then. (i.e. we'll have to deprecate it and remove it, and removing it would require a major version bump.)

If we call it just "aggregation" or "variable_aggregation" (to make it a little less generic), we would meet semver with just adding the new option, and we can note in the documentation how "experimental" (or not) it is.

What do you think?